src/main/java/com/perso/sports/controller/UserController.java

```diff
@@ -1,42 +1,42 @@
 package com.perso.sports.controller;
 
-import com.perso.sports.entity.User;
+import com.perso.sports.entity.UserEntity;
 import com.perso.sports.repository.UserRepository;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 import java.util.Map;
+import java.util.UUID;
 
 @RestController
 public class UserController {
 
     @Autowired
     UserRepository userRepository;
 
-    @GetMapping("/users")
+    @GetMapping("/user")
```

Comment:
Comme vu ensemble on peut laisser un "s" dans l'url
```suggestion
    @GetMapping("/users")
```